openedx/core/djangoapps/bookmarks/views.py

```diff
@@ -219,25 +224,33 @@ def post(self, request):
         POST /api/bookmarks/v1/bookmarks/
         Request data: {"usage_id": "<usage-id>"}
         """
+        default_user_message = ugettext_noop(u'An error has occurred. Please try again.')
+
         if not request.data:
-            return self.error_response(ugettext_noop(u'No data provided.'))
+            return self.error_response(ugettext_noop(u'No data provided.'), default_user_message)
 
         usage_id = request.data.get('usage_id', None)
         if not usage_id:
-            return self.error_response(ugettext_noop(u'Parameter usage_id not provided.'))
+            return self.error_response(ugettext_noop(u'Parameter usage_id not provided.'), default_user_message)
 
         try:
             usage_key = UsageKey.from_string(unquote_slashes(usage_id))
         except InvalidKeyError:
             error_message = ugettext_noop(u'Invalid usage_id: {usage_id}.').format(usage_id=usage_id)
             log.error(error_message)
-            return self.error_response(error_message)
+            return self.error_response(error_message, default_user_message)
 
         try:
             bookmark = api.create_bookmark(user=self.request.user, usage_key=usage_key)
         except ItemNotFoundError:
             error_message = ugettext_noop(u'Block with usage_id: {usage_id} not found.').format(usage_id=usage_id)
             log.error(error_message)
+            return self.error_response(error_message, default_user_message)
+        except BookmarksLimitReachedError:
+            error_message = ugettext_noop(
+                u'You can create up to {0} bookmarks. You must remove some bookmarks before you can add new ones.'
+            ).format(settings.MAX_BOOKMARKS_PER_COURSE)
+            log.error(error_message)
```

Comment:
This is not really an error. `log.info` should do. Also we don't log user facing messages. How about "Attempted to create more than X bookmarks in <course_id>".